docs/Observability/2. prometheus.md

```diff
@@ -0,0 +1,20 @@
+# Prometheus method
+## Introduction
+Prometheus is an open-source monitoring and alerting toolkit designed for reliability and scalability. It collects metrics from configured targets at specified intervals, stores them in a time-series database, and provides powerful querying capabilities. With its flexible architecture, Prometheus is particularly suited for dynamic environments, making it a popular choice for cloud-native applications and microservices. Its intuitive visualization tools help teams gain deep insights into system performance, ensuring timely detection of issues.
+## Configuration
+### AI DIAL Core Settings
+Add the following environment variable to AI DIAL Core configuration. Refer to [AI DIAL Core](https://github.com/epam/ai-dial-helm/tree/main/charts/dial-core) to learn more.
+```json
+containerPorts.metrics
+```
+### AI DIAL Extentions Settings
+Add the following environment variable to AI DIAL Extentions configuration. Refer to [AI DIAL Extentions](https://github.com/epam/ai-dial-helm/tree/main/charts/dial-extension) to learn more.
+```json
```

Comment:
metrics.enabled
metrics.serviceMonitor.enabled

The default port for collecting metrics in most dial applications is 9464. If necessary, you can change the parameter. `containerPorts.metrics`

Reply:
Done